lib/switchman_inst_jobs/switchman/shard.rb

```diff
@@ -120,8 +120,7 @@ def delayed_jobs_shards
           scope = scope.or(::Switchman::Shard.unscoped.where(id: db_jobs_shards)) unless db_jobs_shards.empty?
 
           if has_self
-            self_dbs = ::Switchman::DatabaseServer.all
-                                                  .select { |db| db.config[:delayed_jobs_shard] == "self" }.map(&:id)
+            self_dbs = ::Switchman::DatabaseServer.select { |db| db.config[:delayed_jobs_shard] == "self" }.map(&:id)
```

Comment:
this needs to be reverted (and add a `rubocop:disable` comment -- I know exactly which cop prompted you to do this). While modern switchman allows `select` directly on `DatabaseServer` (a feature I added specifically so I didn't have to ignore the rubocop-rails cop!), older versions that we need to maintain compatibility with do not. `DatabaseServer` is _not_ an ActiveRecord model.